index.html

```diff
@@ -189,8 +189,31 @@ <h3>skills</h3>
             </div>
 
         </div>
+
+        <form
+        action="https://formspree.io/f/mwkydlpq"
+        method="POST"
+        class="form"
+        >
+        <div class="form-des">
+            <h2>Conatct Me</h2>
+            <h3>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </h3>    
+        </div>
+
+        <input type="text" name="name" required maxlength="30" placeholder="Name...">
+
+        <input type="email" name="email" required placeholder="Email...">
+
+        <textarea class="form-text" name="message" maxlength="500" required placeholder="write your message here..."></textarea>
+
+        <!-- your other form fields go here -->
+        <button class="form-button" type="submit">Get in Touch</button>
```

Comment:
- [ ] kindly add button interactions to this button like hover, focus, etc .which will enhance the user's interaction.

Reply:
@mugishasam123  thanks for the suggestion